plugins/chat/assets/javascripts/discourse/components/chat-footer.gjs

```diff
@@ -30,7 +32,10 @@ export default class ChatFooter extends Component {
   }
 
   get shouldRenderFooter() {
-    return this.includeThreads || this.directMessagesEnabled;
+    return (
+      this.chatStateManager.hasPreloadedChannels &&
+      (this.includeThreads || this.directMessagesEnabled)
```

Comment:
We might not need this here? I think channels should always exist if chat is enabled, if so then we only care about threads and direct messages.

Reply:
The thing is we need channels to know if threads are enabled, so if we dont do this we will show the footer without threads button for a second